inst/tests/other.Rraw

```diff
@@ -766,3 +766,10 @@ if (loaded[["nanotime"]]) {
   # respect dec=',' for nanotime, related to #6446, corresponding to tests 2281.*
   test(31, fwrite(data.table(as.nanotime(.POSIXct(0))), dec=',', sep=';'), output="1970-01-01T00:00:00,000000000Z")
 }
+
+# test for bug#6607
+local({
```

Comment:
- no need for :::
 - you filter to one NAME, somewhere redundant to then test that value

Reply:
I think you're using local() to reduce the scope of what gets included in tables() right? 

if so I'd either
- emphasize that in a comment (using local is otherwise unusual in our suite)
- assign to a specific new.env() for clarity

Reply:
Sounds good. I'll make these changes.